auction.py

```diff
@@ -186,14 +190,14 @@ async def _nominate(self, ctx):
 
   @commands.command()
   async def nominate(self, ctx):
-    if self.machine.state == 'starting':
-      self.machine.nom_from_start()
+    # if self.machine.state == 'starting':
+    #   self.machine.nom_from_start()
         
-    elif self.machine.state == 'bidding':
-      return
+    # elif self.machine.state == 'bidding':
+    #   return
     
-    elif self.machine.state == 'nominating':
-      pass
+    # elif self.machine.state == 'nominating':
+    #   pass
```

Comment:
why is this commented out? with git it's considered bad form to comment stuff, either delete it or keep it, with git you'll be able to bring it back later so there's no need to "comment in case we need it later". in this case i think delete it

Reply:
Deleted comments